server/protocols/protocols-imap4/pom.xml

```diff
@@ -99,6 +99,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>${james.protocols.groupId}</groupId>
+            <artifactId>protocols-api</artifactId>
+            <scope>test</scope>
+            <type>test-jar</type>
+        </dependency>
```

Comment:
[ERROR] Failed to execute goal com.github.ekryd.sortpom:sortpom-maven-plugin:2.12.0:verify (strict-pom-ordering) on project james-server-protocols-imap4: The file /home/jenkins/jenkins-agent/workspace/james_ApacheJames_PR-788/server/protocols/protocols-imap4/pom.xml is not sorted -> [Help 1]

Please use `mvn sortpom:sort`